app/redwood.toml

```diff
@@ -18,8 +18,6 @@
     'SENTRY_ORG',
     'SENTRY_PROJECT',
     'EMAIL_PASSWORD',
-    'DISCORD_TOKEN',
-    'DISCORD_CHANNEL_ID',
```

Comment:
Not needed, these are for ENV vars that are needed in the web app, not the api, but to be fair, there are probably a few here I need to remove as well.

Reply:
Thanks - still figuring out the redwood-specific configs :)